### ISSUE TITLE
Day 1 fire, difference between maps and csv files.

### ISSUE THREAD

**Issue by mslucash** [LANDIS-II-Foundation/Extension-SCRPPLE#109]
There seems that there are some inconsistencies between the SCRPPLE maps and log files.

For example, the day-of-fire maps each year show there being a fire on Jan 1st (the maps have a value of one, indicating a fire on julian day 1) over nearly the entire landscape, except for areas that would burn later in the year (Maybe those areas also burned on Jan1, but just burned again later in the year and so it overwrote with new day-of-fire values?). This is despite there being a Spring Start set for April 1st. The fire intensity maps show intensities of 1 across the same area (ie most of the landscape), indicating low-intensity fire. And looking at biomass maps between years, there are also declines in biomass in these areas. 

Contrary to this, the ignition-type maps show the areas that seem to have burned on Jan 1 as having a value of 1 (meaning active and unburned). And when looking at the events-log file, there are no fires listed as having happened on January 1st, only the fires that happen on days later in the year. 

So in summary, when we run SCRPPLE, we keep getting a low-intensity fire that happens on first day of the year across the entire landscape. It looks like these fires are indeed occurring, because of the declines in biomass, fire intensity, and day-of-fire maps, but they're not being recorded in the ignition type map or the log files, and they shouldn't be happening on Jan 1st in the first place.

**Comment by rmscheller** [comment]
I think this is a case of bad documentation.  A value of 1 in the day-of-fire map indicates active and unburned.

L539: https://github.com/LANDIS-II-Foundation/Extension-SCRPPLE/blob/master/src/PlugIn.cs

It is otherwise not possible for there to be a fire without an event log entry (L395-409, same file) but double check my code.  Is there a logical sequence that I'm missing?

**Comment by mslucash** [comment]
We have widespread declines in biomass. In year 1, we have 240 burned cells in year 1 (0.2% of our landscape), but we have declines in biomass in 60% of our cells for paper birch (as an example). There is some loss due to longevity of PB, but I calculated that to be 4406 cells or 4.7% of the landscape. 

![image](https://user-images.githubusercontent.com/7819936/103320120-0aa93100-49e9-11eb-806d-b94739dd7f8d.png)

**Comment by rmscheller** [comment]
Yes, but are we sure that's caused by fire?  Could it be NECN/DGS?
